core/templates/dev/head/components/summary_tile/CollectionSummaryTileDirective.js

```diff
@@ -46,11 +51,21 @@ oppia.directive('collectionSummaryTile', [function() {
         };
 
         $scope.getCollectionLink = function() {
-          return UrlInterpolationService.interpolateUrl(
-            COLLECTION_VIEWER_URL, {
-              collection_id: $scope.getCollectionId()
+          if (angular.isDefined($scope.inCreatorDashboard)) {
+            if ($scope.inCreatorDashboard) {
+              return UrlInterpolationService.interpolateUrl(
+                COLLECTION_EDITOR_URL, {
```

Comment:
From the creator dashboard, on clicking a collection it should go towards the page of editing the collection IMO. I think this was the behaviour before so I decided it would be better to stick to it. What are your opinions on this?